.github/ISSUE_TEMPLATE/breaking_change.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Breaking Change
+about: Track plans for a breaking change
+title: "Removal of ? in v2."
+labels: "breaking change"
+assignees: ""
+---
+
+## Broken API or pattern
+
+<!-- Name the APIs involved specifically, etc. -->
+
+**Preparation References**
+
+<!-- A URL to the deprecation release notes. Alternatively, describe actions required where broken API or pattern is in use. -->
+
+**Packages**
+
+<!-- A simple list of impacted packages -->
+
+**Expected Timeline**
+
+<!-- When you expect this API/pattern will be fully removed/unsupported. If possible, give a release version. -->
+
+**Reminders**
+
+<!-- Reminder: Assign this issue to somebody, if you are unsure who, assign it to yourself. -->
+
+<!-- Associate PR: link a PR from test/... branch in this repo (not in a fork). -->
```

Comment:
What's the reason for the PR from test/ in upstream and not from a fork?

Reply:
So that we can run our integration pipelines. For the legacy-alpha breaks we'll want to run Loop integration for sure. So far that means use a branch under test/ in the microsoft/FluidFramework repository.

Reply:
I get the point, but I bet this is going to be a spot where people continually miss it.  I have spent time thinking through these steps a lot recently, and I still (due to muscle memory) created my branch in my fork the first time.

An alternative workflow is to push to a test branch ad hoc when it's time to run the pipeline, like `git push upstream my-branch:test/foo`.  That's easy enough to do and document.

Also, who do you expect to be running the CI pipeline?  Each breaking change author?  I haven't seen that called out as a requirement or recommendation.

Reply:
This is not the only instructions. I have process pending for the wiki, but I want to say use the breaking change template; so, I need this first. This is more or a reminder because yes people will miss things.